### DISCUSSION TITLE
Batched transforms

### DISCUSSION THREAD

**Discussion by edgarriba** [bitshifter/glam-rs#539] [Q&A] [answered]
hi! I was wondering what is the glamorous way to perform let's say a batched transform e.g A x = b where A is 3x3 and x is Nx3. Thanks ! nice library

**Comment by bitshifter** [comment, marked as answer]
Converted to a discussion since it's not a bug / feature request. :)

glam doesn't really offer anything special for batching.

It may be worthwhile using `Mat3A` and `Vec3A` because they use explicit SIMD but it may or may not be worthwhile depending on how you load and store your data, you'd have to benchmark it to see if it helped.

**Reply by edgarriba** [reply]
ok, thanks for the answer
